src/Criteria/Criteria.php

```diff
@@ -49,10 +49,35 @@ public function toQuery(): string
         return $sql;
     }
 
+    public function toQuerySearch(): string
+    {
+        $sql = '';
+        if (!empty($this->filters)) {
```

Comment:
early return instead of nested code

Reply:
if (empty(...)) {return the computed sql}

Reply:
Done this. I hope I understood well what you meant.

Reply:
yes :)